wix/Main.openj9.wxs

```diff
@@ -38,27 +38,29 @@
 
     <!-- Define the directory structure -->
     <Directory Id="TARGETDIR" Name="SourceDir">
-      <?if $(env.Platform)=x86?>
-				<Directory Id="ProgramFilesFolder">
-          <Directory Id="PRODUCTMANUFACTURER" Name="$(var.ProductManufacturer)">
-            <Directory Id="INSTALLDIR" Name="$(var.AppFolder)" />
-          </Directory>
-				</Directory>
-			<?else?>
-				<Directory Id="ProgramFiles64Folder">
-          <Directory Id="PRODUCTMANUFACTURER" Name="$(var.ProductManufacturer)">
-            <Directory Id="INSTALLDIR" Name="$(var.AppFolder)" />
-          </Directory>
-				</Directory>
-			<?endif?>
+    <?if $(env.Platform)=x86?>
```

Comment:
does this need to be `x86-32`?

Reply:
no because Platform is internally used by Wix and must be x86
and I set it before
`    IF %%G == x86-32 (
        SET PLATFORM=x86
    )`